components/dashboard/src/projects/NewProject.tsx

```diff
@@ -322,7 +371,7 @@ export default function NewProject() {
 
     return (<div className="flex flex-col w-96 mt-24 mx-auto items-center">
         <h1>New Project</h1>
-        <p className="text-gray-500 text-center text-base">Select a git repository.</p>
+        <p className="text-gray-500 text-center text-base">Select a git repository on <strong>{provider}</strong>.</p>
```

Comment:
**question**: Worth linking to the provider itself? Feel free to ignore if it does not seem helpful.

Reply:
**fyi**: Opened follow up issue in https://github.com/gitpod-io/gitpod/issues/5333